database/model/exp.js

```diff
@@ -0,0 +1,124 @@
+
+const hrtime = require('process.hrtime');
+const client = require('../');
+
+let arrayOfValues = [];
+
+const buildNewExpArray = (newExp) => {
+  arrayOfValues = [
+    newExp.title,
+    newExp.city,
+    newExp.state,
+    newExp.country,
+    newExp.category,
+    newExp.activity,
+    parseInt(newExp.averageRating, 10),
+    parseInt(newExp.numberOfReviews, 10),
+    parseInt(newExp.duration, 10),
+    parseInt(newExp.groupSize, 10),
+    newExp.includes,
+    newExp.cuisine,
+    newExp.hostedLanguages,
+    parseInt(newExp.costPerPerson, 10),
+    newExp.imageUrls,
+    newExp.videoUrl,
+  ];
+
+  return arrayOfValues;
+};
+
+const getExp = (expId, callback) => {
+  const selectQuery = {
+    text: 'SELECT * FROM experiences WHERE experienceId = $1',
+    values: [expId],
+  };
+
+  const timer = hrtime();
+  client.query(selectQuery, (err, res) => {
+    if (err) {
+      callback(err);
+    } else {
+      console.log(`Execution time: ${hrtime(timer, 's')}`);
```

Comment:
Remember to comment out logs like this (that will slow down your system) when testing.

Reply:
Thank you!